doc/source/deployment/verify.rst

```diff
@@ -17,3 +17,143 @@ services are functioning as expected by running the following commands:
    export OS_CLOUD='openstack'
    openstack endpoint list
    openstack server list
+
+OpenStack Tempest Testing
+=========================
+
+After the deployment of SUSE Containerized OpenStack has completed, it is possible to run
+OpenStack Tempest tests against the core services in the deployment using the `run.sh` script.
+Before running Tempest tests, it will be necessary to manually configure OpenStack network
+resources and provide a few configuration parameters in the ${WORKDIR}/env/extravars file.
+
+Setting Up An External Network And Subnet in OpenStack
+------------------------------------------------------
+
+To set up an external network and subnet in OpenStack, the following commands can be run from a
+shell on the `Deployer` node.
+
+.. code-block:: console
+
+   export OS_CLOUD=openstack
+   openstack network create --provider-network-type flat --provider-physical-network public \ 
+     --external ext-net
+   openstack subnet create --network ext-net --subnet-range 192.168.100.0/24 --allocation-pool \
+     start=192.168.100.10,end=192.168.100.200 --gateway 192.168.100.1 --no-dhcp ext-subnet
```

Comment:
Same here we can match the example command to the name public-subnet which is default instead of the ext-subnet.